### ISSUE TITLE
armv7 failing to find release

### ISSUE THREAD

**Issue by SelfhostedPro** [ranisalt/node-argon2#376]
### Steps to reproduce
1. Try to build docker container with arm support using buildx
2. It should fail when getting to the section where argon2 is downloaded with a 404 error.
3. Action with logs available here: https://github.com/SelfhostedPro/yacht-api/actions/runs/4848340232/jobs/8639369035

### Expected behaviour
For the package to be available and downloads without issue.

### Actual behaviour
Fails with the following error:
```
#39 [linux/arm/v7 base  8/14] RUN pnpm -r fetch
#39 324.2 .../argon2@0.30.3/node_modules/argon2 install: node-pre-gyp ERR! install response status 404 Not Found on https://github.com/ranisalt/node-argon2/releases/download/v0.30.3/argon2-v0.30.3-napi-v3-linux-arm-musl.tar.gz 
#39 324.2 .../argon2@0.30.3/node_modules/argon2 install: node-pre-gyp WARN Pre-built binaries not installable for argon2@0.30.3 and node@18.16.0 (node-v108 ABI, musl) (falling back to source compile with node-gyp) 
#39 324.2 .../argon2@0.30.3/node_modules/argon2 install: node-pre-gyp WARN Hit error response status 404 Not Found on https://github.com/ranisalt/node-argon2/releases/download/v0.30.3/argon2-v0.30.3-napi-v3-linux-arm-musl.tar.gz 
#39 325.0 .../node_modules/vue-demi postinstall: Done
```

### Environment
**Operating system**:
linux/armv7
Github Actions Runner
BuildX

**Node version:**
18 alpine lts (pnpm)

**Comment by adamwdennis** [comment]
Im also seeing this behaviour... it appears the latest release did not build/publish all the targets of previous releases.

**Comment by 524c** [comment]
The same happened for me with the arm64 architecture.

```
.../argon2@0.30.3/node_modules/argon2 install: node-pre-gyp info check checked for "/app/node_modules/.pnpm/argon2@0.30.3/node_modules/argon2/lib/binding/napi-v3/argon2.node" (not found)
.../argon2@0.30.3/node_modules/argon2 install: node-pre-gyp http GET https://github.com/ranisalt/node-argon2/releases/download/v0.30.3/argon2-v0.30.3-napi-v3-linux-arm64-glibc.tar.gz
```

**Comment by ranisalt** [comment]
@SelfhostedPro is your device running 32-bit ARM? I see that it tries to load for architecture `arm` rather than `arm64`

@adamwdennis indeed, I'll try to trigger the builds again. Github Actions is spectacularly inconsistent

@524c that is not an error, that is just a normal info message

**Comment by ranisalt** [comment]
@524c I noticed issues with the publishing of ARM64 builds for version 0.31.0 due to a bump in the minimum Node version (14 to 16)

**Comment by 524c** [comment]
> @524c I noticed issues with the publishing of ARM64 builds for version 0.31.0 due to a bump in the minimum Node version (14 to 16)

If I can help with anything.. thank you for your support.

**Comment by ranisalt** [comment]
Alpha versions released under the `next` tag will not have issues with missing binary downloads. You can try it by installing `argon2@next` with your preferred package manager, the rest of the library remains the same

**Comment by ranisalt** [comment]
Should be fixed with v0.40.1
